tests/rptest/tests/polaris_catalog_smoke_test.py

```diff
@@ -51,3 +63,75 @@ def test_creating_catalog(self):
 
         assert len(resp.catalogs) == 1
         assert resp.catalogs[0].name == "test-catalog"
+
+    @cluster(num_nodes=2)
+    def test_connectiong_to_catalog(self):
+        """The very basic test checking interaction with polaris catalog
+        """
+        polaris_api = PolarisDefaultApi(self.polaris.management_client())
+        catalog_name = "test-catalog"
+        principal_name = "test-user"
+        principal_role_name = "test-user-role"
+        catalog_role_name = "test-catalog-role"
+        catalog = Catalog(
+            type="INTERNAL",
+            name=catalog_name,
+            properties=CatalogProperties(
+                default_base_location=
+                f"file://{PolarisCatalog.PERSISTENT_ROOT}/catalog_data",
+                additional_properties={}),
+            storageConfigInfo=StorageConfigInfo(storageType="FILE"))
+
+        # create a catalog
+        polaris_api.create_catalog(CreateCatalogRequest(catalog=catalog))
+
+        # create principal and grant the catalog role
+        r = polaris_api.create_principal(
+            CreatePrincipalRequest(principal=Principal(name=principal_name)))
+        credentials = r.credentials
+
+        r = polaris_api.create_principal_role(
+            CreatePrincipalRoleRequest(principalRole=PrincipalRole(
+                name=principal_role_name)))
+
+        r = polaris_api.create_catalog_role(
+            catalog_name=catalog_name,
+            create_catalog_role_request=CreateCatalogRoleRequest(
+                catalogRole=CatalogRole(name=catalog_role_name)))
+        for p in [
+                CatalogPrivilege.CATALOG_MANAGE_ACCESS,
+                CatalogPrivilege.CATALOG_MANAGE_CONTENT,
+                CatalogPrivilege.CATALOG_MANAGE_METADATA,
+                CatalogPrivilege.NAMESPACE_CREATE,
+        ]:
+            r = polaris_api.add_grant_to_catalog_role(
+                catalog_name=catalog_name,
+                catalog_role_name=catalog_role_name,
+                add_grant_request=AddGrantRequest(
+                    grant=CatalogGrant(type='catalog', privilege=p)))
+        r = polaris_api.assign_principal_role(
+            principal_name=principal_name,
+            grant_principal_role_request=GrantPrincipalRoleRequest(
+                principalRole=PrincipalRole(name=principal_role_name)))
+
+        r = polaris_api.assign_catalog_role_to_principal_role(
```

Comment:
were you intending to do something with each of the `r`'s assigned to, like test for an error or something?

Reply:
i checked if the APIs are returning any meaningful results but they don't